zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/job/ActivatableJobsPushTest.java

```diff
@@ -240,33 +249,30 @@ private Long createJob(
     return jobRecord.getKey();
   }
 
-  private void assertEventOrder(final Intent... eventOrder) {
-    for (final long piKey : activeProcessInstances) {
-      final var processInstanceRecordStream = records().betweenProcessInstance(piKey);
-      assertThat(processInstanceRecordStream)
-          .extracting(Record::getIntent)
-          .containsSequence(eventOrder);
-    }
-  }
-
-  private void assertJobActivations(final int activationCount) {
-    final List<Record<JobBatchRecordValue>> batchRecord =
-        RecordingExporter.jobBatchRecords(JobBatchIntent.ACTIVATED).withType(jobType).asList();
-    assertThat(batchRecord).hasSize(activationCount);
+  private void assertEventOrder(
```

Comment:
Hey @korthout, I kept this check in the utility test method as the assertion structure was repetitive.
I've also included explanatory comments above its usage in some places to provide a wider context.